frontend/build-and-copy.sh

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-FROM=./result
+RESULT=./result
+OUT=./out
 TO=./../api/src/treestatus_api/static/ui
 
-rm -f $FROM
-nix-build -o $FROM
+rm -f $RESULT
+rm -rf $OUT
+mkdir $OUT
+docker container rm treestatus || :
+docker run --name treestatus -v $(pwd):/src nixos/nix sh -c "cd /src && nix-build -o $RESULT"
+# pipe through tar to avoid permission denied issues
+docker container cp treestatus:$(readlink -f $RESULT)/ - | tar -C $OUT --strip-components=1 -x
+sudo chown $(id -un): -R $OUT
+chmod 644 $OUT/*
 
 rm -f $TO/*
-cp $FROM/* $TO/ -R
-sudo chown $(id -un): $TO -R
-chmod +rw $TO -R
-
-
+cp -R $OUT/* $TO/
```

Comment:
Maybe a message reminding the user to commit the files in `$TO` now.

Reply:
good point! another commit incoming